src/libraries/System.Runtime/tests/System/AppContext/AppContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public partial class AppContextTests
+    {
+        [Theory]
+        [InlineData("AppContext_Case1", 123)]
+        [InlineData("AppContext_Case2", "")]
+        [InlineData("AppContext_Case3", null)]
+        public void AppContext_GetSetDataTest(string dataKey, object value)
```

Comment:
```suggestion
        public void GetSetDataTest(string dataKey, object value)
```
AppContext is in the test type name. No need to duplicate in the test method.